frontend/src/components/Chat/ChannelInviteDialog.vue

```diff
@@ -82,15 +82,14 @@
       },
     },
     async created() {
-      let response = await axios.get('/users/');
-      for (let i: number = 0; i < response.data.data.length; i++) {
-        this.users.push({
-          id: response.data.data[i].id,
-          username: response.data.data[i].username,
-        });
-        console.log(response.data.data[i].username);
-      };
-      console.log('got users');
+      await axios.get('/users/').then((response) => {
+        for (let i: number = 0; i < response.data.data.length; i++) {
+          this.users.push({
+            id: response.data.data[i].id,
+            username: response.data.data[i].username,
+          });
+        };
+      });
```

Comment:
```suggestion
try {
      await axios.get('/users/');
    
        for (let i: number = 0; i < response.data.data.length; i++) {
          this.users.push({
            id: response.data.data[i].id,
            username: response.data.data[i].username,
          });
        };
      });
} catch (e) {
      console.error(e.message);
}
```
The change is very similar to the original code (except for the `console.log`). I think that using `.then` does not help with exceptions being thrown. You either have to had after the `.then` a `.catch` or just await like in my suggestion.